### DISCUSSION TITLE
RMSE on energies and forces on training set

### DISCUSSION THREAD

**Discussion by douniashaabankabakibo** [ACEsuit/mace#1150] [Q&A]
Hello,
Is there a way to report the RMSE on energies and forces on training set after each epoch (not only on validation set) in the logging file?
Thanks!
Dounia

**Comment by vue1999** [comment]
Hi Dounia, 
Currently, you can only check the training loss in the .txt log files written to the results folder — these contain the losses computed on the training batches. 
You can see this in the plots too. This you can also create during training with the -plot_frequency=N flag, which generates .png files after every N epochs. This way you can track both training and validation sets as training progresses.

**Reply by douniashaabankabakibo** [reply]
Thank you for the quick response! I had a look at the plots. There is one indeed for the training and validation losses, but the plotin the upper right corner  is RMSE on energy and forces only for validation right?
<img width="746" height="442" alt="Screenshot 2025-08-27 at 5 32 33 PM" src="https://github.com/user-attachments/assets/7742e07e-778e-4eda-ae22-a7b733002834" />

**Reply by vue1999** [reply]
Yes, that's correct.

**Reply by douniashaabankabakibo** [reply]
Do you know of a way to get the same plot in the upper right corner for the training set?

**Reply by vue1999** [reply]
At the moment, we don’t have plans to add such feature. However, it should be possible to adapt your local version to include the RMSEs either in the text output or in the plot.
